editor/src/components/editor/store/store-deep-equality-instances.ts

```diff
@@ -591,6 +591,8 @@ export function SpecialSizeMeasurementsKeepDeepEquality(): KeepDeepEqualityCall<
     const clientWidthResult = oldSize.clientWidth === newSize.clientWidth
     const clientHeightResult = oldSize.clientHeight === newSize.clientHeight
     const parentFlexDirectionResult = oldSize.parentFlexDirection === newSize.parentFlexDirection
+    const display = oldSize.display === newSize.display
```

Comment:
I think `displayEquals` would be better. or go with the pattern and `displayResult` but I like equals better because this is not a comparison result object